demo-notebooks/additional-demos/hf_interactive.ipynb

```diff
@@ -317,14 +315,13 @@
     "assert ray_cluster_uri, \"Ray cluster needs to be started and set before proceeding\"\n",
     "\n",
     "import ray\n",
-    "from ray.air.config import ScalingConfig\n",
     "\n",
     "# reset the ray context in case there's already one. \n",
     "ray.shutdown()\n",
     "# establish connection to ray cluster\n",
     "\n",
     "#install additional libraries that will be required for this training\n",
-    "runtime_env = {\"pip\": [\"transformers\", \"datasets\", \"evaluate\", \"pyarrow<7.0.0\", \"accelerate\"]}\n",
+    "runtime_env = {\"pip\": [\"transformers\", \"datasets==2.17.0\", \"pyarrow>7.0.0\", \"accelerate\", \"scikit-learn\"]}\n",
```

Comment:
Can these be all pinned? Is `pyarrow` a direct dependency?

Reply:
I can pin them if that's better. 
A direct dependency of the sdk? No, just needed to run this training script. Apologies if I misunderstand what you are asking.

Reply:
Ok looked at it again, have removed pyarrow as it is already installed with sdk dependencies and have pinned the others. Thanks @astefanutti :)